src/prepend_title_cell.py

```diff
@@ -0,0 +1,55 @@
+import json
+import nbformat as nbf
+import argparse
+import sys
+
+def add_ipynb_title(title: str, input_file: str, output_file: str, replace_first=False):
+    """Add or replaces the first cell of an ipython notebook with a markdown title
+
+    title (str): Markdown text to reside in the first cell
+    input_file (str): Path to the input file or template to be modified
+    output_file (str): Path to the write destination
+    replace_first (bool): If true, replaces the first cell of the input_file instead of prepending
+        a new cell
+
+    CLI use example:
+
+    python prepend_title.py -t '# New Header' -i 'Metrics Template.ipynb' -o 'my_notebook.ipynb'
+
+    """
+    # Ensure master template file won't be overwritten
+    if output_file.split('/')[-1]== 'Metrics Template.ipynb':
```

Comment:
![windowspath](https://user-images.githubusercontent.com/13056349/59934961-be0b8f80-941a-11e9-9d68-f20ec679ce8f.PNG)

It actually looks like this is natively handled by os.path. However, it will detect what types of paths are being used based on the machine that python is running on. This might be a more graceful way to handle

Reply:
lgtm I'll make that change